### PR TITLE
💬 Status adaption for Nextcloud 20

### PR DESCRIPTION
**Pull request by nickvergessen**
- [x] Add away/dnd state on mentions
- [x] Add away/dnd state on the participant list
- [ ] Show status icon as overlay on avatars? => Follow up after nextcloud-vue support

### Mentions

* Status shown when away or dnd

### Participant list

* Status shown when away or dnd
* People with status sorted to the end of their section (moderator, online, user status)

> Bildschirmfoto von 2020-08-05 17-10-03

### CODE REVIEW

**Comment by nickvergessen** [comment]
Depends on https://github.com/nextcloud/server/pull/22106, but works otherwise

**Comment by nickvergessen** [comment]
Moving the status to a second line in the participant list is out of my knowledge

**Comment by jancborchardt** [comment]
> Moving the status to a second line in the participant list is out of my knowledge

@ma12-co @skjnldsv to the rescue :)

**Comment by nickvergessen** [comment]
Got some help, so it's now on a new line, but it should probably be trimmed in case the user has bad intentions
Bildschirmfoto von 2020-08-05 15-29-06

**Comment by nickvergessen** [comment]
But we can do that as a follow up

File: css/At.scss [hunk 1]
```diff
@@ -150,8 +150,12 @@
 		}
 
 		.atwho-cur {
-			background: var(--color-primary);
+			background: var(--color-primary-element);
```
Comment (jancborchardt) [inline css/At.scss:153]: If we use --color-primary-light, then we don’t have to change the text color, and the status is also distinguished for the person who is selected. That way it also looks consistent with the global style as per Vue components and https://github.com/nextcloud/server/pull/22115

Comment (nickvergessen) [reply css/At.scss:153]: I tried it together with Marco, it is much worse with certain combinations.

Comment (nickvergessen) [reply css/At.scss:153]: nevermind, I used element-light, instead of only light

Comment (nickvergessen) [reply css/At.scss]: fixed

**Pull request merged**